python/lxml-in-pandas.yml

```diff
@@ -0,0 +1,10 @@
+rules:
+- id: lxml-in-pandas
+  patterns:
+    - pattern: pandas.read_html(...)
+    - pattern-not: pandas.read_html(..., flavor="bs4", ...)
+    - pattern-not: pandas.read_html(..., flavor="html5lib", ...)
```

Comment:
What about `**kwargs`, will it be caught? If not, document it pls

Reply:
It catches it.